File: messageservicejs/app/models/conversation.js
const mongoose = require('mongoose');

const conversationSchema = new mongoose.Schema({
    conversationId: {
        type: String,
        required: true,
    },
    firstUserId: { 
        type: String,
        required: true,
    },
    secondUserId: {
        type: String,
        required: true,
    }
}, { 
    versionKey: false,
});

const Conversation = mongoose.model('Conversation', conversationSchema);

module.exports = Conversation;

File: front_service/public_flutter/lib/models/conversation.dart
class Conversation {
  final String id;
  final String firstUserId;
  final String secondUserId;

  Conversation({
    required this.id,
    required this.firstUserId,
    required this.secondUserId,
  });

  factory Conversation.fromJson(Map<String, dynamic> json) {
    return Conversation(
      id: json['_id'],
      firstUserId: json['firstUserId'],
      secondUserId: json['secondUserId'],
    );
  }
}